axiom/response.go

```diff
@@ -6,6 +6,8 @@ import "net/http"
 type Response struct {
 	*http.Response
 
+	TraceID string
```

Comment:
Since we embed the *http.Response, and we can get the TraceID from the header, I think it would be nicer to just add a method to Response like this:

```go
func (r *Response) TraceID() string {
    return r.Header.Get(headerTraceID)
}
```